aad/azure-spring-boot-sample-active-directory-b2c-resource-server/README.md

```diff
@@ -0,0 +1,140 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-active-directory-b2c
+description: "Sample project for Azure AD B2C Spring Boot resource server client library"
+urlFragment: "azure-spring-boot-sample-active-directory-b2c-resource-server"
```

Comment:
Could you please share related code about using this?

Reply:
related issue: https://github.com/Azure/azure-sdk-for-java/issues/20718
the generated code will be [here ](https://github.com/MicrosoftDocs/samples/tree/master/Azure/azure-sdk-for-java)